src/main/webapp/about.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Page Title</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDbqn_ZFgoyKA_oDqFDC1vJyBq7Z3m0SgM"></script>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+<div id="nav-container">
+    <a href="index.html" class="nav-link">Home</a>
+    <a href="listings.html" class="nav-link">Browse listings</a>
+    <a href="about.html" class="nav-link">About us</a>
+    <div id="search-container">
+        <form action="/placeholder.js">
+            <input type="text" placeholder="Search listing...">
+            <button type="submit"><i class="fa fa-search"></i></button>
+        </form>
+    </div>
+</div>
```

Comment:
Consider using HTML templating to add our nav bar to every page, rather than the raw html to each .html. There are multiple approaches listed [here](https://stackoverflow.com/questions/44502243/single-navigation-bar-across-website) and [this W3 page](https://www.w3schools.com/howto/howto_html_include.asp). 

This way we can avoid having to change HTML in every single page every time we update the navbar.

Reply:
Cool, thanks for the suggestion